index.js

```diff
@@ -889,10 +889,8 @@ let createContainer = ({resolvers = [], dependencies = {}, factories, definition
 
         if (!definition.reuse) {
             destroyObject(instance, options);
-            definition.instance = null;
-        } else {
-            delete definition.instance;
         }
+        definition.instance = null;
```

Comment:
I see that when we are using reuse then after `delete definition.instance` there still will be `instance` property of reused module. But why do we need it?